examples/custom_collector/main.go

```diff
@@ -7,7 +7,7 @@ import (
 	"os/signal"
 	"syscall"
 
-	"github.com/etf1/kafka-transformer/pkg/transformer/kafka"
+	transformer2 "github.com/etf1/kafka-transformer/pkg/transformer"
```

Comment:
I'm not a huge fan of the `tranformer2` import ;)
Maybe you can create a `factory` package including the `NewKafkaTransformer` function.
What do you think ?

Reply:
I'll look to change the names of the packages.
Transformer interface and KafkaTransformer can be ambiguous